api/types.proto

```diff
@@ -359,6 +359,15 @@ message TaskStatus {
 	}
 }
 
+message AttachmentStatus {
```

Comment:
Reusing task status for this is unwise. If task status represents this resource, then we should use tasks. Although, I think we need a resource allocation model.